School.js

```diff
@@ -29,20 +35,20 @@ class School {
    * @return {Student} Enrolled student
    */
   enrollStudent(className, student) {
-    // Your code here
+    console.log(this.classes[className].students.push(student))
```

Comment:
you should `return this.classes[className].students.push(student)` , not console.log. That way you can use the return value in your app.js file.